### DISCUSSION TITLE
keyring issue

### DISCUSSION THREAD

**Discussion by mshzsh** [hyprwm/Hyprland#663] [Q&A] [answered]
in vscode when syncing settings by microsoft or github accounts, i get this error :

`Writing login information to the keychain failed with error 'Error calling StartServiceByName for org.freedesktop.secrets: Timeout was reached'.`

gnome-keyring libsecret libgnome-keyring are alreadey installed

**Comment by vaxerski** [comment]
polkit auth agent?

**Reply by mshzsh** [reply]
yes!

**Reply by vaxerski** [reply]
i mean do you have one, like `polkit-kde-agent`

**Reply by mshzsh** [reply]
No, I don't have any of them installed

**Reply by vaxerski** [reply]
well thats prolly why you have this error

**Reply by mshzsh** [reply]
do i need to install `polkit-dumb-agent-git`?

**Reply by vaxerski** [reply]
no clue. Choose the agent that you want. Only one is needed.

**Reply by mshzsh** [reply]
I installed one of them and tested opening the root directory and it worked fine, but the problem still persists in vscode

**Reply by vaxerski** [reply]
it's not a hyprland issue. You're missing some polkit / keyring deps.

**Reply by mshzsh** [reply]
Yes, the issue is not for hyperland, I will try harder to solve it, thank you

**Comment by mshzsh** [comment, marked as answer]
I replaced `emptty` display manager  with `ly` and the problem was solved
